### ISSUE TITLE
Error building a program against reckless

### ISSUE THREAD

**Issue by npezolano** [mattiasflodin/reckless#8]
I was able to build the library with the make file, however when you try to compile the examples or include reckless in another project the following error is thrown:

```
g++ -std=c++11 severity_log.cpp -I/home/n/git/cpp/reckless/reckless/include -L/home/n/git/cpp/reckless/reckless/lib -lreckless
In file included from /home/n/git/cpp/reckless/reckless/include/reckless/policy_log.hpp:25:0,
                 from /home/n/git/cpp/reckless/reckless/include/reckless/severity_log.hpp:25,
                 from severity_log.cpp:22:
/home/n/git/cpp/reckless/reckless/include/reckless/basic_log.hpp:31:43: fatal error: boost_1_56_0/lockfree/queue.hpp: No such file or directory
compilation terminated.
```

**Comment by mattiasflodin** [comment]
You need to also add -I/home/n/git/cpp/reckless/boost to your gcc command line. I'll update the readme to reflect this.

**Comment by mattiasflodin** [comment]
Fixed in c05a9406511. Could you check the documentation at https://github.com/mattiasflodin/reckless/tree/next#building and see if that works for you?

**Comment by mattiasflodin** [comment]
Resolved in c05a940.
